.github/workflows/codeql.yml

```
@@ -0,0 +1,39 @@
+      fail-fast: false
+      matrix:
+        language: [javascript-typescript]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v6
+
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v3
+        with:
+          language: ${{ matrix.language }}
```

actionlint (1.7.12)

[error] input "language" is not defined in action "github/codeql-action/init@v3". available inputs are "analysis-kinds", "build-mode", "check-run-id", "config", "config-file", "db-location", "debug", "debug-artifact-name", "debug-database-name", "dependency-caching", "external-repository-token", "languages", "matrix", "packs", "quality-queries", "queries", "ram", "registries", "setup-python-dependencies", "source-root", "threads", "token", "tools", "trap-caching"

(action)